### ISSUE TITLE
Adding overflowBehaviour='menu' to Pivot component causes accessibility checks to fail

### ISSUE THREAD

**Issue by Onokaev** [microsoft/fluentui#20202]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:
Adding overflowBehaviour='menu' to Pivot component causes accessibility checks to fail. The checks fail because the overflow button that appears lacks an aria-label, button-name or aria-description for accessibility compliance. 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/EvansAboge/pen/LYjVjzG?editors=1010

#### Actual behavior:
The overflowBehaviour works very well on Pivot components and makes the pivot items very responsive. However, the button lacks a button-name or aria-label or aria-description for accessibility compliance

#### Expected behavior:

The overflowBehaviour should also include a prop that supplies a button-name, aria-label or aria-description to the overflow button.

**Comment by gouttierre** [comment]
@Onokaev - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows. 

@khmakoto  - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by Onokaev** [comment]
Hi @gouttierre. It is a new issue brought by the introduction of overflow behavior. 
It was very well done to be honest. It just needs more props for setting aria labels to the overflow menu
